.github/workflows/test.yml

```diff
@@ -0,0 +1,13 @@
+name: test
+
+on:
+  - pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    container: miki725/pre-commit
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Pre-Commit
+        run: pre-commit run --all-files
```

Comment:
Not sure about all files. I think we were omitting some. E.g translations

Reply:
this just lints all files precommit should be checking as per its config. see https://pre-commit.com/#4-optional-run-against-all-the-files

Reply:
ok